examples/connectivity/plot_extract_regions_canica_maps.py

```diff
@@ -0,0 +1,84 @@
+"""
+Regions extraction using Canonical ICA maps and functional connectomes
+======================================================================
+
+This example shows how to use :class:`nilearn.regions.region_extractor.RegionExtractor`
+to extract connected brain regions from whole brain ICA maps and
```

Comment:
Shouldn't the class  be referenced as  nilearn.regions.RegionExtractor? I would like as much as possible not to go more than one level deep.